uber/config.py

```diff
@@ -72,6 +72,72 @@ def PREREG_BADGE_TYPES(self):
         return types
 
     @property
+    def BADGE_DISPLAY_TYPES(self):
+        """
+        There are several contexts where we want to display different badge types to select:
+        - A new attendee registering for the event
+        - An attendee claiming a blank badge in an existing group
+        - An existing attendee editing their registration
+
+        This property uses the parameters in the URL to figure out the context and return the correct
+        badge types to display for an attendee.
+        """
+        with sa.Session() as session:
+            params = cherrypy.lib.httputil.parse_query_string(cherrypy.request.query_string)
```

Comment:
Yeah, if you wanted to have arguments in a Django template you'd need to do that via a custom tag.  The more idiomatic approach would probably be to write this as a method and then have a custom tag which is just a thin wrapper around this function.  For now we should probably just get something working and then look at it once we have it in a good state.  (Side note: if I was picking a template engine today I'd pick Jinja templates, which are basically "We've copied Django templates but you can actually call methods with parameters and such.")